### PR TITLE
ci(.github & build): support build multi-platforms image

### PR DESCRIPTION
**Pull request by Gkirito**
Run ci test on my fork repo [link](https://github.com/Gkirito/ehco/actions)
The multi-platforms [image](https://hub.docker.com/repository/docker/gkirito/ehco/tags?page=1&ordering=last_updated)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -64,13 +67,14 @@ jobs:
...
         uses: docker/build-push-action@v4
         with:
           context: .
           tags: "ehco1996/ehco:latest"
           push: true
           file: build/Dockerfile
+          platforms: linux/amd64,linux/arm64
 
       # TODO fix the arm build error  gcc: error: unrecognized command-line option '-marm'; did you mean '-mabm'?
       # - name: Build arm image
```
Comment (Ehco1996) [inline .github/workflows/ci.yml:80]: how about  remove those todos in this pr?

**Review by Ehco1996 — COMMENTED** [review]
rest lgtm

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Gkirito) [reply .github/workflows/ci.yml:80]: 👌

**Review by Ehco1996 — APPROVED** [review]
lgtm nice job !

**Pull request merged**